### DISCUSSION TITLE
AmazonChime; Status Code: 400; Error Code: ValidationException

### DISCUSSION THREAD

**Discussion by Thalita-Martins** [aws/aws-sdk-java-v2#4044] [General]
We use Amazon Chime to create online rooms, but in some requests the AmazonChime error is being returned; Status Code: 400; Error Code: ValidationException, analyzing the documentation of the CreateAttendee and CreateMeeting endpoints that are called to create a room with a participant I couldn't find any information about what this error could be. 

The calls are coming back with error logs:

2023-05-24 16:35:44.705 ERROR 1 --- [nio-8080-exec-9] .f.f.c.CustomDataFetcherExceptionHandler : Exception while fetching data (/createAttendeeBySujeitoAtencao) : Unable to unmarshall exception response with the unmarshallers provided (Service: AmazonChime; Status Code: 400; Error Code: ValidationException; Request ID: a0a8c9f9-54d7-4da1-9520-e4c1bfd80c8f; Proxy: null)
com.amazonaws.AmazonServiceException: Unable to unmarshall exception response with the unmarshallers provided (Service: AmazonChime; Status Code: 400; Error Code: ValidationException; Request ID: a0a8c9f9-54d7-4da1-9520-e4c1bfd80c8f; Proxy: null)

2023-05-24 16:35:40.669 ERROR 1 --- [nio-8080-exec-3] .f.f.c.CustomDataFetcherExceptionHandler : Exception while fetching data (/createAtendeeBySujeitoAtencao) : Unable to unmarshall exception response with the unmarshallers provided (Service: AmazonChime; Status Code: 400; Error Code: ValidationException; Request ID: 6ddd93751-84a3-45ff-8094-b6c0f86c945e; Proxy: null)
com.amazonaws.AmazonServiceException: Unable to unmarshall exception response with the unmarshallers provided (Service: AmazonChime; Status Code: 400; Error Code: ValidationException; Request ID: 6ddd93751-84a3-45ff-8094-b6c0f86c945e; Proxy: null)

Could you please forward us the requests and the logs of these Request ID mentioned in the text so we can evaluate and verify what error is being sent.

**Comment by debora-ito** [comment]
@Thalita-Martins can you share a code sample I can use to reproduce the error?
Which version of the SDK are you using?

**Reply by Thalita-Martins** [reply]
We are using sdk 2.17.178 and the code implementation. Would it be possible to forward me the requests that appear in the error log so I can check how the request is being forwarded. Below is the Id of the request with errors:

Request ID: e3cb2e45-2655-4eda-86b9-b6f2d73fb3a6

Request ID: c5fb0023-f38f-4aec-8405-a57c0a0e7914

Request ID: cbad64cd-e172-40c9-90d2-a6445a4ef7ec

[exemplo.da.implementacao.txt](https://github.com/aws/aws-sdk-java-v2/files/11705793/exemplo.da.implementacao.txt)

**Reply by debora-ito** [reply]
We, the SDK team, don't keep request logs, only the service team has them.

The code sample you sent was a snippet of the SDK code. What I asked for was a sample of your code where you are using the SDK Chime client, so I can run the same sample code on my side and try to reproduce the error you are seeing. See http://sscce.org/ for details on how to provide a sample code.

**Reply by Thalita-Martins** [reply]
@debora-ito I will request the code from the Chime SDK client to perform the test. In the case how do I contact the service team to request these log records of these requests with errors?

**Reply by debora-ito** [reply]
If you have access to a AWS Support Plan you can provide the request IDs to the Support team and they will contact the service team that maintains the respective APIs. But in this case I would try to reproduce the error using the SDK first, because "Unable to unmarshall" is a client-side error. I'll wait for you to provide a sample code so I can try to repro.

In the meantime, you can enable the verbose wirelogs and see if the response contains additional information. Please refer to our [Dev Guide](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/logging-slf4j.html#sdk-java-logging-verbose) for instructions on how to enable the verbose wirelogs. If you choose to provide the logs to us, please make sure to redact any sensitive information (like access keys).
